src/main/java/com/demo/classroom/Service/MyUserDetails.java

```diff
@@ -0,0 +1,66 @@
+package com.demo.classroom.Service;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import com.demo.classroom.Entity.User;
+import com.demo.classroom.Repository.StudentRepository;
+import com.demo.classroom.Repository.TeacherRepository;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class MyUserDetails implements UserDetails {
```

Comment:
What is the purpose of creating MyUserDetails class while we can do this even in the UserDetails class?

Reply:
Yes we can do that. I you think its better to move the class to UserService class. I can do that. I just want to make sure the class are independent. This classs will be needed in our future enhancement.

Reply:
Merge UserDetails and MyUserDetails

Reply:
done